Improve Documentation - II (#1383)
SpikeInterface/spikeinterface: doc/modules/sortingcomponents.rst

    see also [here](https://openreview.net/forum?id=ohfi44BZPC4)
  * 'monopolar_triangulation' with optimizer='minimize_with_log_penality'

> Theses methods are the same as implemented in :py:mod:`spikeinterface.postprocessing.unit_localization`

Reviewer: ```suggestion
These methods are the same as implemented in :py:mod:`spikeinterface.postprocessing.unit_localization`
```

**Notes**:

>   * :code:`spatial_interpolation_method` "krigging" or "iwd" fot not play a big role.

Reviewer: ```suggestion
  * :code:`spatial_interpolation_method` "kriging" or "idw" donot seem to play a big role.
```

>   * :code:`border_mode` is very important details. How to deal with the border because with motion units on the border

Reviewer: ```suggestion
  * :code:`border_mode` is a very important detail. It controls how to deal with the border because with motion units on the border
```

Author: I tried rewriting this one for clarity so definitely give it a look over to see if you like how I rewrote it.
